test/alternator/test_streams.py

```diff
@@ -769,11 +766,74 @@ def do_updates(table, p, c):
         return events
     do_test(test_table_ss_old_image_and_lsi, dynamodbstreams, do_updates, 'OLD_IMAGE')
 
-# TODO: test NEW_IMAGE and NEW_AND_OLD_IMAGES, similar to OLD_IMAGE above.
-# In fact, it can be exactly the same code, sharing one do_update() function.
-# Just make sure to keep three separate tests on the three separate tables -
-# testing NEW_AND_OLD_IMAGES is not enough - OLD_IMAGE might have a bug,
-# e.g., a missing column, that isn't missing in NEW_AND_OLD_IMAGES.
+# Tests similar to the above tests for OLD_IMAGE, just for NEW_IMAGE mode.
```

Comment:
Avi already committed my patch adding these tests to next. We'll need to either wait for master to contain it and rebase over master - or if you want this in quickly, please rebase over next now.
In either case, don't forget after the rebase to delete the "xfail" I added in my patch.